base-image/Dockerfile

```diff
@@ -144,8 +144,10 @@ ONBUILD USER ${NB_USER}
 # We want to keep our images as reproducible as possible. If a lock
 # file with exact versions of all required packages is present, we use
 # it to install packages. conda-lock (https://github.com/conda-incubator/conda-lock)
-# is used to generate this conda-linux-64.lock file from a given environment.yml
-# file - so we get the exact same versions each time the image is built. This
+# is used to generate this conda-lock.yml file from a given environment.yml
+# file - so we get the exact same versions each time the image is built. Since this is
+# different packages for different CPU architectures
+# the same dockerfile can be used to build different architecture images. This
```

Comment:
Sentence read a little funny. I guess we're trying to say that while different CPU architectures will require packages that are compiled differently, we can still use the multi-platform conda-lock.yml file for the Docker build?

```suggestion
# file - so we get the exact same versions each time the image is built. Note that
# different packages may be used for different CPU architectures, but still,
# the same dockerfile can be used to build different architecture images. This
```